src/features/Apiexplorer/RequestResponseRenderer/index.tsx

```diff
@@ -58,7 +59,7 @@ const PlaygroundSection = <T extends TSocketEndpointNames>({
     <div
       id='playground-console'
       className={style.playgroundConsole}
-      data-testid='playgroundConsole'
+      data-testid='playground-section'
```

Comment:
we should prevent using hard coded `data-testid`, but if there are no other ways, it's ok, thanks man :pray:

Reply:
don't think it can be tested another way because the content is different for api call